scripts/check_license.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
Should this file itself have a license header, btw? :D

Reply:
There isn't one in the other shell scripts, nor other supporting files (e.g. Dockerfile). As a) I'm probably less of an expert than Rob that wrote this stuff, and b) consistency, I'm strongly inclined not to start adding stuff here.

Reply:
I'm inclined to agree.